code/modules/clothing/gloves/color.dm

```diff
@@ -169,22 +169,38 @@
 
 /obj/item/clothing/gloves/color/latex
 	name = "latex gloves"
-	desc = "Cheap sterile gloves made from latex."
+	desc = "Cheap sterile gloves made from latex. Transfers minor paramedic knowledge to the user via budget nanochips."
 	icon_state = "latex"
-	item_state = "lgloves"
+	item_state = "latex"
 	siemens_coefficient = 0.3
 	permeability_coefficient = 0.01
 	item_color="mime"
 	transfer_prints = TRUE
 	resistance_flags = NONE
+	var/carrytrait = TRAIT_QUICK_CARRY
```

Comment:
Do you need var for this?

Reply:
If you don't use a var, then there should be another set of overrides for `/obj/item/clothing/gloves/color/latex/nitrile` which actually will be 3 times more messier.

Reply:
right, but it should at least follow naming scheme and be called carry_trait